rts/at/annotation/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jboss.narayana.quickstart.rts</groupId>
+    <artifactId>rts-quickstarts</artifactId>
```

Comment:
What about replacing these line with :
```
    <groupId>org.jboss.narayana.quickstart.rts</groupId>
    <artifactId>annotation</artifactId>
    <version>6.0.2.Final-SNAPSHOT</version>
    ```
    and remove the 'relativePath' ?

Reply:
Don't forget that it needs a rebase.